qt/scientific_interfaces/test/ISISReflectometry/Batch/BatchPresenterTest.h

```diff
@@ -26,6 +26,7 @@ using testing::NiceMock;
 using testing::Return;
 using testing::ReturnRef;
 using testing::_;
+using testing::_;
```

Comment:
Has a duplicate line been added by mistake?

Reply:
Probably an issue with the rebase, I shall remove it.